pkg/utils/nodeclaim/suite_test.go

```diff
@@ -161,7 +163,7 @@ var _ = Describe("NodeClaimUtils", func() {
 
 		providerID := test.RandomProviderID()
 		nodeClaim.Status.ProviderID = providerID
-		Expect(nodeclaimutil.UpdateStatus(ctx, env.Client, nodeClaim)).To(Succeed())
+		Expect(env.Client.Status().Update(ctx, nodeClaim)).To(Succeed())
```

Comment:
If we're dropping these functions, then these tests aren't really testing anything anymore so I think that we can drop them